test/models/photo_test.rb

```
@@ -0,0 +1,23 @@
+require "test_helper"
+
+class PhotoTest < ActiveSupport::TestCase
+  def setup
+    @user = User.create!(email: "photo_test_#{Time.now.to_i}@example.com", display_name: "Photo Tester", password: "password")
```

RuboCop (1.75.5)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)